server/svix-server/src/db/models/messagecontent.rs

```diff
@@ -0,0 +1,55 @@
+// SPDX-FileCopyrightText: © 2022 Svix Authors
+// SPDX-License-Identifier: MIT
+
+use crate::core::types::MessageId;
+use chrono::Utc;
+use sea_orm::entity::prelude::*;
+use sea_orm::ActiveValue::Set;
+
+#[derive(Clone, Debug, PartialEq, Eq, DeriveEntityModel)]
+#[sea_orm(table_name = "messagecontent")]
+pub struct Model {
+    #[sea_orm(primary_key, auto_increment = false)]
+    pub id: MessageId,
+    pub created_at: DateTimeWithTimeZone,
+    pub payload: Vec<u8>,
+    pub expiration: DateTimeWithTimeZone,
+}
+
+#[derive(Copy, Clone, Debug, EnumIter, DeriveRelation)]
+pub enum Relation {
+    #[sea_orm(
+        belongs_to = "super::message::Entity",
+        from = "Column::Id",
+        to = "super::message::Column::Id",
+        on_update = "NoAction",
+        on_delete = "Cascade"
```

Comment:
Wonder how this works without the FK to connect the two tables.
I _thought_ cascade deletes required a trigger if you didn't have a constraint in place.

Reply:
I'm sure you're right. This was mostly a copy-paste of other models which generally have the same setting, and IIRC, we don't have _any_ FK constraints in the DB.

Reply:
Removed entirely, since it's pointless anyway.